### DISCUSSION TITLE
Temperature and Humidity Occupancy - How does it work?

### DISCUSSION THREAD

**Discussion by barryperfect** [Hankanman/Area-Occupancy-Detection#12] [Q&A]
First of all, great idea for an Occupancy Integration!

I have a room that currently just has a temperature and humidity sensor in it and I am trying to avoid having to put any other sensors in.

How does the temperature and humidity occupancy work? Is it just looking for them to increase by a certain amount to indicate the room is occupied and when they go down indicate not occupied, or something else?

**Comment by Hankanman** [comment]
The idea is that it would find a baseline from your sensor history and the probability changes when the current values are outside that baseline, so for example the average humidity if you bathroom might be 60% but when you get in the shower it shoots up to 90%. I need to add some more diagnostic and config options for environmental so you can tweak it on the fly

**Reply by barryperfect** [reply]
Thanks for that, let me know if there is anything I can do to help test that functionality

**Comment by mickek** [comment]
Why not add CO2 sensors to the environment sensors? Those values are directly correlated with presence and can be quite reliable

**Comment by mickek** [comment]
And PM2 for kitchen / cooking
